### PR TITLE
Unban Zygarde-10% from [Gen 7] Pokebank UU (alpha)

### PR DESCRIPTION
**Pull request by panpawn**
http://www.smogon.com/forums/threads/sun-moon-uu-speculation-and-discussion-thread-see-post-85-for-uu-alpha.3587419/page-4#post-7126546

### CODE REVIEW

**Comment by panpawn** [comment]
(Under Hikari's instruction)

**Comment by panpawn** [comment]
DO NOT MERGE, THIS IS SUPPOSED TO BE Zygarde-10%

**Comment by panpawn** [comment]
Okay, I'm confused - when I go in the teambuilder with a Zygarde-10% and validate, it says:
>- Zygarde is banned by [Gen 7] Pokebank UU (alpha).

But, I wasn't using a Zygarde in my team - I was using a Zygarde-10%.  How do I unban one but not the other?

**Comment by panpawn** [comment]
Okay, I think sirDonovan helped me figure this out now.

File: config/formats.js [hunk 1]
```diff
@@ -56,8 +56,8 @@ exports.Formats = [
 			'Bisharp', 'Buzzwole', 'Celesteela', 'Chansey', 'Clefable', 'Dugtrio', 'Excadrill', 'Ferrothorn', 'Garchomp', 'Genesect',
 			'Greninja', 'Heatran', 'Hippowdon', 'Hoopa-Unbound', 'Kartana', 'Kingdra', 'Landorus', 'Latios', 'Magearna', 'Magnezone',
 			'Manaphy', 'Mandibuzz', 'Mantine', 'Marowak-Alola', 'Mimikyu', 'Nihilego', 'Pelipper', 'Pheromosa', 'Porygon-Z', 'Rotom-Wash',
-			'Skarmory', 'Tapu Bulu', 'Tapu Fini', 'Tapu Koko', 'Tapu Lele', 'Toxapex', 'Tyranitar', 'Xurkitree', 'Zapdos', 'Zygarde',
-			'Alakazite', 'Charizardite X', 'Gyaradosite', 'Metagrossite', 'Pinsirite', 'Sablenite', 'Scizorite', 'Venusaurite',
+			'Skarmory', 'Tapu Bulu', 'Tapu Fini', 'Tapu Koko', 'Tapu Lele', 'Toxapex', 'Tyranitar', 'Xurkitree', 'Zapdos', 'Zygarde-base',
+			'Zygarde-100%', 'Alakazite', 'Charizardite X', 'Gyaradosite', 'Metagrossite', 'Pinsirite', 'Sablenite', 'Scizorite', 'Venusaurite',
```
Comment (Zarel) [inline config/formats.js]: This should be `Zygarde-Complete`

Comment (TheImmortal) [inline config/formats.js]: Also ``Base`` for consistency sake

Comment (panpawn) [inline config/formats.js]: Done and done

**Pull request merged**